scripts/config-jetstream-darkmode.json

```diff
@@ -0,0 +1,220 @@
+{
+  "source": ["./src/themes/jetstream/color-darkmode/*.json"],
+  "include": [
+    "./src/themes/jetstream/color/base.json",
+    "./src/themes/jetstream/color/brand.json",
+    "./src/themes/jetstream/comments.json"
+  ],
+  "platforms": {
```

Comment:
For now we should only be exporting the CSSCustomProperties platform for jetstream (including darkmode)

Reply:
Sure.

Reply:
@fajar-apri-alaska The JSONVariables are not needed as part of the JetStream theme. That format is part of the Excursion theme but not the JetStream theme.